Citizen-Science/src/screens/SignupScreen.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { useState } from 'react';
+import { ImageBackground, SafeAreaView, Text, TouchableOpacity, Dimensions} from "react-native";
+import styles from '../StyleSheet';
+import CustomButton from '../components/CustomButton';
+import CustomTextField from '../components/CustomTextField';
+import { useNavigation } from '@react-navigation/native';
+import {NativeStackNavigationProp} from '@react-navigation/native-stack';
+import LogoName from "../components/LogoName";
+import BackButton from "../components/BackButton";
+import ScreenTitle from "../components/ScreenTitle";
+import homeScreen from "./HomeScreen";
+import loginScreen from "./LoginScreen";
+import {responsiveHeight, responsiveWidth} from "react-native-responsive-dimensions";
+import Blobs from "../components/Blobs";
+
+type NavigationParam = {
+    Login: undefined;
+    Signup: undefined;
+    NavigationBar: undefined;
+};
+
+//Type for our Navigation in our component
```

Comment:
Try to remove any comments like these, a lot of comments in the code are helpful (e.g. TODOs, explanations of complicated code/why it was done a certain way, etc.)

Reply:
Perfect, we'll remove the redundant comments! 💪